### ISSUE TITLE
Training loss explodes when reproducing cvpr in settings

### ISSUE THREAD

**Issue by trThanhnguyen** [Dolorousrtur/HOOD#30]
Dear HOOD's authors,

Thank you for publishing your work, it's fascinating.
I need your help with an issue when trying to reproduce training `cvpr_submission.pth` on my own: the training loss explodes. As the attached image shows, the training is very unstable, the total loss starts to explode since 55K iteration at magnitude ~5e7, and it only keeps getting worse when I keep training until 150K iters as described in the paper. I understand this is not expected behavior, when I evaluate the checkpoints from 60K to 150K, all of them yield catastrophic results.

For the training settings, I follow your exact recipe in the repo: from data downloading to code, config, etc. 

Can you please share your thoughts on this? Was the described settings used to train the cvpr's model? Your help would be invaluable. Thank you very much.

<img width="6528" height="3328" alt="Image" src="https://github.com/user-attachments/assets/d25f6818-3966-4401-a630-acfed30f8abe" />
